src/publish-tab/publish-tab.js

```diff
@@ -0,0 +1,181 @@
+import $ from 'jquery'
+import DefineMap from 'can-define/map/map'
+import Component from 'can-component'
+import template from './publish-tab.stache'
+
+const LHICallback = function guideZipped (data) {
```

Comment:
this is my bad, but these functions should not be named `guideZipped` for all of them, should just be dropped as the `const` declaration covers the name. copy-paste mistake